src/app/utils/IsEphemerallyDeployed.ts

```diff
@@ -0,0 +1,15 @@
+import type { Machine } from "app/store/machine/types";
+import { NodeStatusCode } from "app/store/types/node";
+
+/**
+ * Check if a machine is deployed ephemerally or not
+ * @param machine - A machine object
+ * @returns boolean value
+ */
```

Comment:
I appreciate that you tried to follow existing conventions and we do use JSDoc extensively in the utils file.

On the other hand, this JSDoc doesn't provide any additional information beyond what's already apparent from the TypeScript types and name of this function, and only adds noise.

We should prioritize clear, self-explanatory code, and only use JSDoc to provide context or additional information that cannot be inferred from the code itself.

```suggestion
```

Reply:
Added info about this to our docs: https://github.com/canonical/maas-ui/pull/5156